### PR TITLE
part of fix for 'add jsonnet tests for all libsonnet files'

### PR DESCRIPTION
**Pull request by kkasravi**
part of fix for #1542

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1835)

### CODE REVIEW

**Comment by ashahba** [comment]
Thanks @kkasravi for updating the test cases.
/approve

File: kubeflow/core/prototypes/ambassador.jsonnet [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // @description Ambassador Component
 // @shortDescription Ambassador
 // @param name string Name
-// @optionalParam cloud string null Cloud
+// @optionalParam platform string none supported platforms {none|gke|minikube}
```
Comment (jlewi) [inline kubeflow/core/prototypes/ambassador.jsonnet]: Was @ashahba 's PR already submitted? Why change "null" to "none"? Is that to match what kfctl does?

Comment (kkasravi) [reply kubeflow/core/prototypes/ambassador.jsonnet]: @ashahba's PR missed optionalParam. Yes - changed to 'none' for kfctl.

**Pull request merged**